charts/posthog/templates/plugins-deployment.yaml

```diff
@@ -13,7 +13,9 @@ spec:
         app: {{ template "posthog.fullname" . }}
         release: "{{ .Release.Name }}"
         role: plugins
+  {{- if not .Values.plugins.hpa.enabled }}
```

Comment:
Have you tried out the autoscaling on plugins?

I don't think the hpa will work as-is on it - from what I've seen in the past it's not really cpu-bound even when it's falling behind.

Reply:
Nope, didn't try it, only changed some parts about it (because I was confused about what I saw across various values files, so I did some reading). I'd love to look into this more, but I suspect it's not a priority atm. The alternative I considered was removing everything tied to autoscaling from this repo, but given that we likely want to add something about it in the future I opted not to.